Makefile

```diff
@@ -1,6 +1,11 @@
+ifeq ($(CIRCLE_TAG),)
+  TAG = PRTEST
+else
+  TAG = ${CIRCLE_TAG}
+endif
```

Comment:
idea:

```
CIRCLE_TAG =? PRTEST
TAG = ${CIRCLE_TAG}
```
rather than the logical operators?

whether or not this is clearer, you be the judge 😄

Reply:
Yep, I tried that operator first, but it doesn't work.  According to the GNU docs `?=` does a comparison to `undefined`, which this value is not (it's empty string).  The `ifeq` is a bit more flexible with falsey values, it seems.

Reply:
Actually, the way you have it, it might work.

Reply:
Ah, I didn't do it that way so as not to mess with Circle's env vars within here.  That felt like a no-no mutation.

Reply:
re-working based on @jorgebay's suggestion